CHANGELOG.md

```diff
@@ -18,6 +18,16 @@
  - Fix virus count sorting on "Virus Report"
  - Remove additional slashes in "SpamAssassin Rule Description Update"
  - Adding translation to user interface
+ - Add LOGO path in conf.php.example and changes done in corresponding files.
+ - layout changes in quarantine.php, other.php, sf_version.php
+ - Removed duplicate PHP function in tools/Cron_jobs/quarantine_report.php (is in functions.php)
+ - Changes in tools/Sendmail_relay/INSTALL
+ - Changes in tools/Sendmail_relay/sendmail_relay.init
```

Comment:
I mean: could you explain what changes has be done in tools/Sendmail_relay/INSTALL and tools/Sendmail_relay/sendmail_relay.init, so that the casual reader/installer know if he has to take action or not.